Modules/Module_Processing/Module_Population/xASL_wrp_CreatePopulationTemplates.m

```diff
@@ -716,10 +716,10 @@ function xASL_wrp_CreatePopulationTemplates4Sets(x, bSaveUnmasked, bRemoveOutlie
         try
             WithinGroup = SetID==UniqueSet(iU);
 
-            if ~SessionsExist(iScanType) && length(WithinGroup)==x.dataset.nSubjectsSessions
+            if ~SessionsExist(iScanType) && length(WithinGroup) == x.dataset.nSubjects * x.dataset.nSessions
                 % if no sessions exist, but "WithinGroup" definition was
                 % based on all subject/sessions, then correct this
-                CurrSess = [1:length(listSessions):x.dataset.nSubjectsSessions]' + (iSession-1);
+                CurrSess = (1:length(listSessions):x.dataset.nSubjects * x.dataset.nSessions)' + (iSession-1);
```

Comment:
These brackets need to return to [ ] square brackets, because this is a list. At least, this was on purpose, not a mistake

Reply:
That was not necessary. [] brackets are needed for a list, but 1:2:4 is a list by itself and it doesn't need more brackets. So you only need normal brackets before the transposition.

The following is all correct:
[1,2,3]
1:3
[1:2,3]